raphtory-graphql/src/model/graph/graph.rs

```diff
@@ -22,13 +22,15 @@ use std::{collections::HashSet, convert::Into};
 #[derive(ResolvedObject)]
 pub(crate) struct GqlGraph {
     name: String,
+    namespace: Option<String>,
```

Comment:
are we possibly overcomplicating this? Could we not just store the `namespace/name` as the `name` and be done here, no need for all the options and all that?